pkg/ambex/main.go

```diff
@@ -489,14 +489,14 @@ func update(
 		ecp_v3_resource.RuntimeType:  runtimesv3,
 	}
 
-	snapshotv3, err := ecp_v3_cache.NewSnapshot(version, snapshotResources)
+	snapshot, err := ecp_v3_cache.NewSnapshot(version, snapshotResources)
```

Comment:
Yes, these were dropped in Emissary V3 because as of Envoy 1.22 it no longer supported the v2 xds Protocols. We only support v3 at this time.

Emissary v2.3.z still supports both v2 and v3.